Plugins/Encounter/Encounter.cpp

```diff
@@ -192,12 +199,78 @@ ArgumentStack Encounter::SetResetTime(ArgumentStack&& args)
     if (auto *pEncounter = encounter(args))
     {
         auto resetTime = Services::Events::ExtractArgument<int32_t>(args);
-          ASSERT_OR_THROW(resetTime >= 0);
+        ASSERT_OR_THROW(resetTime >= 0);
 
         pEncounter->m_nResetTime = resetTime;
     }
 
     return Services::Events::Arguments();
 }
 
+ArgumentStack Encounter::GetNumberOfSpawnPoints(ArgumentStack&& args)
+{
+    int32_t retVal = 0;
```

Comment:
Wouldn't `-1` be a better error return value, since 0 is a valid value?

Reply:
Yes it would, but it's contrary to 99% of the NWN stock functions that don't let you differenciate 0 from ERROR.

Reply:
For example: Encounter::GetResetTime returns 0 on error. It could return -1 to indicate error.